app/views/components/_document-list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  group_index ||= false
+  track_clicks = "data-module=track-click" if group_index
+  dimension28 = items.count.to_s
+%>
+<ol class="app-c-document-list" <%= track_clicks %>>
+  <% items.each_with_index do |link, link_index| %>
+    <li class="app-c-document-list__item">
+      <h3 class="app-c-document-list__item-title">
+        <%
+          data = {}
+          if group_index
+            data = {
+              track_category: 'navDocumentCollectionLinkClicked',
+              track_action: "#{group_index + 1}.#{link_index + 1}",
+              track_label: link[:base_path],
+              track_options: {
+                dimension28: dimension28,
+                dimension29: link[:title]
+              }
+            }
+          end
+        %>
+        <%=
+          link_to(
+            link[:title],
+            link[:base_path],
+            data: data
+          )
+        %>
+      </h3>
+      <ul>
+        <li class="app-c-document-list__attribute">
+          <%
+            document_updated_at = link[:public_updated_at]
+            if not document_updated_at.is_a?(ActiveSupport::TimeWithZone)
+              document_updated_at = ActiveSupport::TimeZone['UTC'].parse(document_updated_at)
```

Comment:
I wondered about this - whether to go with GMT and try to switch to BST or something complex. It seemed like a rabbit hole and then I realised that only the date is used, not the time.